CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+* fix(events): fix display for events new_autoscaler and edit_autoscaler in timeline [#684](https://github.com/Scalingo/cli/pull/684)
```

Comment:
There is also the `repeated_crash` event that we now support. And this changelog entry should be mve in the "To be Released section

Reply:
Is it possible to link 2 issues to a single PR ?
Like "Fix #681 and #682"